test/sixpack-test.js

```diff
@@ -26,6 +28,27 @@ describe("Sixpack in node", function () {
         }
     });
 
+    it("should forward 'Cookie' header", function (done) {
+        var http = require('http');
+        var originalGet = http.get;
+        var receivedHeaders;
+        try {
+          http.get = (url, options, callback) => {
```

Comment:
You should look for this indentation

Reply:
Aaargh, you're right, it's mixing `\s\s` with `\s\s\s\s`. I'll fix it, thank you for noticing! 8)